docs/api/access-control.md

```diff
@@ -8,14 +8,14 @@ order: 4
 
 Control who can do what with your GraphQL API.
 
-> **Note:** This is the API documentation for access control. For getting started, see the [Access control guide](/docs/guides/access-control.md) or the [Authentication Guide](/docs/guides/authentication.md).
+> **Hint:** This is the API documentation for access control. For getting started, see the [Access control guide](/docs/guides/access-control.md) or the [Authentication Guide](/docs/guides/authentication.md).
```

Comment:
This is totally bike-shedding but why hint over note? 

FWIW, I really don't think this is a hint... but whatever

Reply:
Sure, let's change it back 🙂